asreview/webapp/src/HomeComponents/DashboardComponents/ProjectTable.js

```diff
@@ -342,8 +342,8 @@ const ProjectTable = (props) => {
                           variant="subtitle1"
                           noWrap
                         >
-                          {row["datetimeCreated"]
-                            ? formatDate(row["datetimeCreated"])
+                          {row["created_at_unix"]
+                            ? formatDate(row["created_at_unix"])
```

Comment:
Nice. 

One day we should look at property names I think. Google's style guide is nice. https://google.github.io/styleguide/jsoncstyleguide.xml#Property_Name_Guidelines